pkg/config/config_template.yaml

```diff
@@ -21,6 +21,19 @@ api_key:
 #
 # dd_url: https://app.datadoghq.com
 
+## @param cloud_provider - string - optional - default: <empty>
+## This option restrict which endpoint will be used by the agent to retrieve host metadata. By default the agent will try
```

Comment:
nit: actually not only host metadata is retrieved from these endpoints (but sometimes `metadata` more generally)

```suggestion
## This option restrict which cloud provider endpoint will be used by the agent to retrieve metadata. By default the agent will try
```

Reply:
made the recommended changes here, thanks!